### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by hulchak**
https://hulchak.github.io/layout_search-bar-airbnb/
https://hulchak.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir-Heavy;
```
Comment (AlisaVlasova) [inline src/style.css:2]:
```suggestion
  font-family: Avenir;
```
it should be the same font-family, but different  font-weight

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-size: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-size: 300;
```
Comment (AlisaVlasova) [inline src/style.css:12]:
```suggestion
  font-weight: 300;
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+form.search-bar {
```
Comment (AlisaVlasova) [inline src/style.css:15]:
```suggestion
.search-bar {
```

Comment (AlisaVlasova) [reply src/style.css:15]: use classes, it will be enough
fix similar cases

File: src/index.html [hunk 4]
```diff
@@ -6,11 +6,54 @@
...
+        src="images/Search.svg"
+        alt="icon search"
+        class="icon"
+      >
+      <input
+        type="search"
+        class="search"
+        data-qa="keypress"
+        placeholder="Try &ldquo;Los Angeles&rdquo;"
+      >
```
Comment (AlisaVlasova) [inline src/index.html:37]: better wrap input and image into label
now image is not clickable

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+.icon.small {
+  position: absolute;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+input.search {
+  width: 100%;
+  box-sizing: border-box;
```
Comment (AlisaVlasova) [inline src/style.css:38]: move box-sizing to * selector

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: block;
+  padding: 0;
+  padding-left: 62px;
+  background: #fff;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```
Comment (AlisaVlasova) [inline src/style.css:47]: move font to html selector

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+input.search.small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+input.search::placeholder {
+  color: #3d4e61;
+}
```
Comment (AlisaVlasova) [inline src/style.css:62]:
```suggestion
input.search::placeholder {
  color: #3d4e61;
  font-weight: 300;
}
```

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+input[type="search"]:hover {
```
Comment (AlisaVlasova) [inline src/style.css:64]:
```suggestion
.search:hover {
```

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input[type="search"]:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
```
Comment (AlisaVlasova) [inline src/style.css:75]:
```suggestion
  font-weight: 800;
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 10]
```diff
@@ -6,11 +6,56 @@
...
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="icon search"
+          class="icon small"
+        >
+        <input
+          type="search"
+          class="search small"
```
Comment (sTorba24) [inline src/index.html:55]:
```suggestion
          class="search search--small"
```
image

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```
Comment (sTorba24) [inline src/style.css:17]: set it for html

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.search-bar {
+  margin-top: 20px;
+  position: relative;
+  background: #fff;
```
Comment (sTorba24) [inline src/style.css:23]: it's default value so you can remove it

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+  outline: none;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: block;
+  padding: 0;
+  padding-left: 62px;
+  background: #fff;
```
Comment (sTorba24) [inline src/style.css:51]: this too

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Good work! Lets improve it

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 14]
```diff
@@ -6,11 +6,56 @@
...
+      data-qa="small"
+      action="/"
+      method="get"
+      class="search-bar search--small"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="icon search"
+          class="icon search--small"
```
Comment (tetiana-ivanchenko) [inline src/index.html:51]:
```suggestion
          class="icon icon--small"
```

File: src/index.html [hunk 15]
```diff
@@ -6,11 +6,56 @@
...
+          placeholder="Try &ldquo;Los Angeles&rdquo;"
+        >
+      </label>
+    </form>
+
+    <form
+      data-qa="small"
+      action="/"
+      method="get"
+      class="search-bar search--small"
```
Comment (tetiana-ivanchenko) [inline src/index.html:45]:
```suggestion
      class="search-bar search-bar--small"
```

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**